docs/content/guides/accessibility/accessibility.md

```diff
@@ -0,0 +1,185 @@
+
+## Conformance with Standards
+Several regulatory standards were established by the governing bodies, for instance the European Union or the United States, which align with the Web Content Accessibility Guidelines (WCAG). They consist of a number of guidelines maintained by the World Wide Web Consortium (W3C). <br>
+Handsontable delivers conformance to W3C standards by meeting the requirements of [WCAG 2.1](https://www.w3.org/WAI/WCAG21/quickref/) at level AA.<br> 
+You may find a comprehensive list of policies appertaining to web accessibility in various jurisdictions [here](https://www.w3.org/WAI/policies/).
+
+- **Europe / EU:**
+  - [European Accessibility Act (EAA)](https://ec.europa.eu/social/main.jsp?catId=1202)
+  - [Web Accessibility Directive (WAD)](https://eur-lex.europa.eu/legal-content/EN/LSU/?uri=CELEX:32016L2102)
+
+- **United States:**
+  - [Section 508 of the US Rehabilitation Act](https://www.section508.gov/)
+  - [Americans with Disabilities Act (ADA)](https://www.ada.gov/resources/web-guidance/)
+
+- **Canada:**
+  - [Standard on Web Accessibility](https://www.tbs-sct.canada.ca/pol/doc-eng.aspx?id=23601)
+
+### Accessibility Statement and Voluntary Product Accessibility Template (VPAT)
+The **Accessibility Statement** is a company's commitment, indicating how well their product caters to individuals with disabilities. **Voluntary Product Accessibility Template (VPAT)**, meanwhile, provides detailed insights into how a product aligns with user-friendly standards, such as the Web Content Accessibility Guidelines (WCAG).
+
+Why do these documents matter? Consider Handsontable. Companies integrating Handsontable into their applications aim for optimal user accessibility. If their apps adhere to standards like WCAG 2.1 AA, they expect Handsontable not to compromise that. By offering an **Accessibility Statement** and **VPAT**, Handsontable assures its commitment to universal usability.
+
+### Accessibility testing
+
+Before releasing a new version of Handsontable, we carefully test it for accessibility, using a combination of different approaches:
+
+- We cover the most common use cases with automated unit and end-to-end tests.
+- We manually test all of Handsontable's features with the most popular screen readers (odniesienie do sekcji screen readers w tym artykule ).
+- We use automated visual regression testing.
+- We check Handsontable's accessibility score with a range of the most popular accessibility testing tools, such as Lighthouse, Axe-core, or Accessibility Insights for Web.
+
+## Keyboard Navigation
+The principle of accessible web components is that users who cannot use a mouse can still navigate the product with a keyboard or other assistive technology.
+
+Handsontable provides intuitive navigation that allows access to all focusable elements in a predictable way. It also identifies elements currently holding focus across cells, headers, and subcomponents such as menus, fly-out editors, and more.
```

Comment:
I wonder if its worth saying our keyboard navigation is conforming to the standards, or is that already obvious?